python/lsst/dax/imgserv/jobqueue/imageworker.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of dax_imgserv.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import tempfile
+import os
+import json
+from configparser import RawConfigParser
+
+from celery import Celery
+
+from flask import current_app
+# use following format for circular reference
+import lsst.dax.imgserv.vo.imageSODA as imageSODA
+
+defaults_file = os.environ.get("WEBSERV_CONFIG", "~/.lsst/webserv.ini")
```

Comment:
It looks like this part that isn't in a function parses out the config file and sets it to a module variable.  Maybe put the logic in a function, and then call that function instead

def parse_config():
...

webserv_config = parse_config() or something.

Reply:
Will streamline webserv_config, by centralizing where it's created into a function.